### DISCUSSION TITLE
Disable Upscale Option on Unlimited

### DISCUSSION THREAD

**Discussion by horse14t** [nagadomi/nunif#125] [Ideas]
Sometimes I find I have an image that's too big for standard Waifu and would like to use Unlimited but it has no option to turn off upscaling if all I want to do is clean up the jpg a bit.

**Comment by nagadomi** [comment]
1x means upscaling is disabled.

**Reply by horse14t** [reply]
Ooohhhhh, I didn't realize that!
Never mind then, thank you 😅
